src/main/java/rape/brutal/gitstarship/ammunition/Gun.java

```diff
@@ -15,6 +15,14 @@
     protected int damage;
     protected String name;
     protected int size;
+    protected int cost;
+
+    public Gun(int damage, String name, int size, int cost) {
```

Comment:
Why is there a damage? Damage gets from ammo, is not it? Or damage gets from ammo and from gun?